juju/js-libjuju
# Add missing facades for juju 3.0

## Description

- Update the `schema.json` to the stable `juju-3.0` branch
- Rerun the `build` task

### QA

- Link the library to juju-dashboard
- Update the imports in `juju/index.js` to the latest version of the facades:
```js

import actions from "@canonical/jujulib/dist/api/facades/action-v7";
import allWatcher from "@canonical/jujulib/dist/api/facades/all-watcher-v3";
import annotations from "@canonical/jujulib/dist/api/facades/annotations-v2";
import applications from "@canonical/jujulib/dist/api/facades/application-v15";
import client from "@canonical/jujulib/dist/api/facades/client-v6";
import cloud from "@canonical/jujulib/dist/api/facades/cloud-v7";
import controller from "@canonical/jujulib/dist/api/facades/controller-v11";
import modelManager from "@canonical/jujulib/dist/api/facades/model-manager-v9";
import pinger from "@canonical/jujulib/dist/api/facades/pinger-v1";
```
- Run Juju dashboard on juju v3.0 stable
- Make sure that the dashboard isn't failingd
